grr/server/grr_response_server/gui/ui/lib/api/api_interfaces.ts

```diff
@@ -626,6 +626,7 @@ export declare interface OsqueryArgs {
   readonly query?: string;
   readonly timeoutMillis?: DecimalString;
   readonly ignoreStderrErrors?: boolean;
+  readonly fileCollectColumns?: ReadonlyArray<string>;
```

Comment:
See my comments above re the naming. If needed, we can also change the protobug field's name (the code is not in master, so that should be fine).